models/articles.model.js

```diff
@@ -93,4 +93,20 @@ function updateArticleById (article_id, inc_votes) {
         })
 }
 
-module.exports = {selectArticleById, updateArticleById, selectArticles}
\ No newline at end of file
+function selectCommentsByArticleId(article_id) {
+    return selectArticleById(article_id)
```

Comment:
We don't need to do two requests to the database here. We can do one query and request all of the comments for an article with the article id provided. If that were to come back as 0, then we would make another request to check if the article actually exists. 

If it does - 200 OK! 
if not - 404 - not found!